api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionSummaryPaths.scala

```diff
@@ -29,18 +40,47 @@ trait SubmissionSummaryPaths
 
   implicit val timeout: Timeout
 
+  case class TsLarSummary(ts: Option[TransmittalSheet], larSize: Int, hmdaFileDetails: HmdaFileDetails)
+
   // institutions/<institutionId>/filings/<period>/submissions/<submissionId>/summary
-  // NOTE:  This is currently a mocked, static endpoint
   def submissionSummaryPath(institutionId: String)(implicit ec: ExecutionContext) =
-    path("filings" / Segment / "submissions" / IntNumber / "summary") { (period, submissionId) =>
+    path("filings" / Segment / "submissions" / IntNumber / "summary") { (period, seqNr) =>
       timedGet { uri =>
+        val submissionId = SubmissionId(institutionId, period, seqNr)
+
         val supervisor = system.actorSelection("/user/supervisor")
+        val submissionManagerF = (supervisor ? FindProcessingActor(SubmissionManager.name, submissionId)).mapTo[ActorRef]
+        val validatorF = (supervisor ? FindProcessingActor(HmdaFileValidator.name, submissionId)).mapTo[ActorRef]
+        val hmdaRawF = submissionManagerF.flatMap(actorRef => (actorRef ? GetActorRef(HmdaRawFile.name)).mapTo[ActorRef])
+
+        val tsF = for {
+          validator <- validatorF
+          hmdaRaw <- hmdaRawF
+          s <- (validator ? GetState).mapTo[HmdaFileValidationState]
+          fileDetails <- (hmdaRaw ? GetFileName).mapTo[HmdaFileDetails]
+          larSize = s.lars.size
+          ts = s.ts
+          tsLarSummary = TsLarSummary(ts, larSize, fileDetails)
+        } yield tsLarSummary
+
+        onComplete(tsF) {
+          case Success(x) => x.ts match {
+            case Some(t) =>
+              val contactSummary = ContactSummary(t.contact.name, t.contact.phone, t.contact.email)
+              val agency = Try(Agency.withValue(t.agencyCode)).getOrElse(Agency.UndeterminedAgency)
+              val respondentSummary = RespondentSummary(t.respondent.name, t.respondent.id, t.taxId, agency.name, contactSummary)
 
-        //To avoid having to deal with relative paths on different systems
-        val summaryJson = "{\n  \"respondent\": {\n    \"name\": \"Bank\",\n    \"id\": \"1234567890\",\n    \"taxId\": \"0987654321\",\n    \"agency\": \"CFPB\",\n    \"contact\": {\n      \"name\": \"Your Name\",\n      \"phone\": \"123-456-7890\",\n      \"email\": \"your.name@bank.com\"\n    }\n  },\n  \"file\": {\n    \"name\": \"lar.dat\",\n    \"year\": \"2016\",\n    \"totalLARS\": 25\n  }\n}"
-        val response = HttpResponse(StatusCodes.OK, entity = HttpEntity(ContentTypes.`application/json`, summaryJson))
+              val fileSummary = FileSummary(x.hmdaFileDetails.name, period, x.larSize)
```

Comment:
Instead of using `period` for the File Summary, could we use `t.activityYear` instead? That would show them info they submitted in their file, rather than info from the API call.